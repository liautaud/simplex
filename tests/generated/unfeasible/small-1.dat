8
4
2 12 10 5 4 -10 -8 28
0 -2 -2 -4
-1 -4 1 -3 0 -4 1 2
-1 1 0 2 0 1 3 3
1 3 1 0 1 -3 -3 3
3 2 4 3 1 3 -3 0
