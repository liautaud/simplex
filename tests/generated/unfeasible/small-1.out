╒═══════════════════════════════════════╕
| Linear solver - OA course - ENSL 2017 |
╘═══════════════════════════════════════╛

The input linear program is:

Maximize  +2_x1  +12_x2 +10_x3 +5_x4  +4_x5  -10_x6 -8_x7  +28_x8 
Such that -1_x1  -4_x2  +1_x3  -3_x4         -4_x6  +1_x7  +2_x8  <= 0
          -1_x1  +1_x2         +2_x4         +1_x6  +3_x7  +3_x8  <= -2
          +1_x1  +3_x2  +1_x3         +1_x5  -3_x6  -3_x7  +3_x8  <= -2
          +3_x1  +2_x2  +4_x3  +3_x4  +1_x5  +3_x6  -3_x7         <= -4
          x_1, x_2, x_3, x_4, x_5, x_6, x_7, x_8 are non-negative

This linear program is UNFEASIBLE.
